elastic/kibana
# [Cloud Security][Quick Wins] Fix for Field Selector issue

## Summary

This PR is to address issue where whenever user tries to check on the field on the field selector outside the 1st page, user will be navigated to the 1st page upon checking the checkbox. 

https://github.com/user-attachments/assets/89375c54-deac-49a7-a3a1-2a4d5745ccc7

Steps to test:
- Get some findings data in
- Navigate to Findings Page
- Group by None
- Click on Fields selector, navigate to other pages 
- Check on any of field checkboxes

Expected Result:
User should not get navigated back to page 1
